fabfile.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import print_function
+
+from fabric.api import env, run
+
+import osm_aux_db
+import osm_db
+import osmaxx
+
+from host_setup import provide_docker_compose
+
+env.use_ssh_config = True
+
+OSM_AUX_COMPONENT = 'osm_aux_db'
+OSM_DB_COMPONENT = 'osm_db'
+OSMAXX2 = 'osmaxx'
+
+ALL_COMPONENTS = {
+    OSM_AUX_COMPONENT: osm_aux_db,
+    OSM_DB_COMPONENT: osm_db,
+    OSMAXX2: osmaxx,
+}
```

Comment:
In retrospect, this is utterly confusing. What are the values, what are the keys?

Maybe this would be more readable as
```python
ALL_COMPONENTS = {osm_aux_db, osm_db, osmaxx}  # a set of modules
ALL_COMPONENT_NAMES = {c.__name__ for c in ALL_COMPONENTS}  # a set of strings
```
or even just as
```python
ALL_COMPONENT_NAMES = {'osm_aux_db', 'osm_db', 'osmaxx'}  # a set of strings
```

`ALL_COMPONENT_NAMES` would then be used where we currently use `set(ALL_COMPONENTS)` or `ALL_COMPONENTS.keys()`.

```python
for component in on_components:
    # ...
    module = ALL_COMPONENTS[component]
```
would instead have to be written as
```python
for component_name in on_components:
    # ...
    module = locals()[component_name]
```
(`assert on_components.issubset(ALL_COMPONENT_NAMES)` would already make sure we don't access arbitrary code by that.)